falafel/qe.py

```diff
@@ -453,3 +453,32 @@ def qe_pointsources(px,mlmax,fTalm,xfTalm=None):
     #spin 0 salm 
     salm=0.5*res[0] 
     return salm
+
+def qe_source(px,mlmax,profile,fTalm,xfTalm=None):
```

Comment:
Looks good, docstring needs a little work (for px and mlmax). Also we could cut down on repeated code by combining `qe_pointsources` and `qe_source` e.g. have `qe_source` do the point-source case when `profile=None` or `profile='src'` or something. Could keep the `qe_pointsources` for backwards compatibility but update it to just call `qe_source(..., profile=None)`.

Reply:
I changed `qe_source` such that when `profile=None` the profile is just an array of ones of `size=mlmax`

Reply:
Also could clarify whether profile is a profile in ell space?